book/website/ethical-research/activism/activism-cultural-change.md

```diff
@@ -0,0 +1,97 @@
+(er-activism-cultural-change)=
+
+# Cultural Change
+
+(er-activism-cultural-change-what)=
+
+## What is cultural change?
+
+“inspiring a change in behaviour in persons/organisations, with a lasting impact, strengthening the core values of persons/organisation
```

Comment:
might be useful up here to describe some situations for cultural change and where researchers might encounter them? Sort of expanding on this with a "why is culture change and associated concepts important for data scientists to understand?" paragraph

Reply:
Working on this - should probably improve but I need to move on for today :( Thanks!